modules/custom/reservation/js/reservation.js

```diff
@@ -36,6 +37,35 @@ function SelectingMovieOnClick() {
 
     }
 
+function controlingModal() {
+    var modals = document.getElementsByClassName("modal");
+    var btns = document.getElementsByClassName("button");
+    var spans=document.getElementsByClassName("close");
+
+    for(let i=0;i<btns.length;i++){
+        btns[i].onclick = function() {
+            modals[i].style.display = "block";
```

Comment:
Izbegavati ovako nesto, odraditi preko stilova iz css-a, pa proveri kako se to postize.

Reply:
Odradjeno